WcaOnRails/spec/factories/team_members.rb

```diff
@@ -1,8 +1,90 @@
 FactoryGirl.define do
   factory :team_member do
-    team_id 1
-    user_id 1
-    start_date "2016-02-18"
+    team
+    user
+    start_date 1.month.ago
     end_date nil
+    after(:build) do |team_member|
+      committee_position = FactoryGirl.create(:committee_position, committee_id: team_member.team.committee_id)
+      team_member.committee_position_id = committee_position.id
```

Comment:
Hi @jonatanklosko,

If I do the following:
`tm = FactoryGirl.build(:team_member)` it doesn't save `tm` to the database so I can't reload.

If I do:

`tm = FactoryGirl.create(:team_member)` then it saves the record so then the assignment is saved. So a reload will work. So I think this is OK? Let me know if this isn't right.